packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_screenshot.spec.ts

```diff
@@ -160,4 +160,163 @@ context('screenshot specs', { experimentalSessionSupport: true, experimentalMult
       cy.wrap(onAfterScreenshot).should('be.called')
     })
   })
+
+  it('supports pausing timers', () => {
+    cy.switchToDomain('foobar.com', [this.serverResult], ([serverResult]) => {
+      cy.stub(Cypress, 'automation').withArgs('take:screenshot').returns(
+        {
+          // TODO: A bluebird promise is expected but we can't require bluebird yet
```

Comment:
I think you might be able to get away using [Cypress.Promise](https://docs.cypress.io/api/utilities/promise#Syntax) to get a bluebird promise.

Reply:
That worked! Thanks! [f2491de](https://github.com/cypress-io/cypress/pull/20209/commits/f2491de2f18a1aace35d1b7a17f544b1c52ed5be)